test/solidity/block/BlockInfo.js

```diff
@@ -0,0 +1,105 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+const { expect } = require('chai')
+const { ethers } = require('hardhat')
+const { BigNumber } = require('ethers');
+const Constants = require('../../constants')
+
+describe('@solidityequiv1 BlockInfo Test Suite', function () {
+  let blockInfo, provider, signers
+
+  before(async function () {
+    signers = await ethers.getSigners()
+    provider = ethers.getDefaultProvider();
+    
+    const factory = await ethers.getContractFactory(Constants.Path.BLOCK_INFO)
+    blockInfo = await factory. deploy({ gasLimit: 15000000 })
+  })
+
+  // EIP-1559 does not apply to Hedera
```

Comment:
Can you clarify intent here.
Hedera explicitly supports EIP 1559 transaction types.

Reply:
Would be better to just say base fee don't adjust per block. The EIP itself does apply

Reply:
Done.